translations/base-tr.yaml

```diff
@@ -658,32 +666,40 @@ storyRewards:
         desc:
             "Bu seviyenin bir ödülü yok ama bir sonrakinin olacak!<br><br> Not: Şu anki fabrikalarını yok etmemeni öneririm
             - Daha sonra <strong>Geliştirmeleri açmak için </strong> <strong>bütün hepsine</strong> ihtiyacın olacak!"
+
     no_reward_freeplay:
         title: Sonrakİ Sevİye
         desc: Tebrikler!
     reward_freeplay:
         title: Özgür Mod
+
         desc: Başardın! <strong>Özgür modu</strong> açtın! Bu artık gelen şekillerin
             <strong>rastgele</strong> oluşacağı anlamına geliyor!<br><br>
             Bundan sonra ana bölge belirli bir miktar eşya değil <strong>belirli bir miktar eşya geliş hızına</strong>
             bağlı olarak level atlayacaksın, istenilen şekilleri otomatik olarak yapacak bir fabrika inşa etmeni
             öneririm!<br><br> Ana bölgenin istediği şekil kablo katmanında sol taraftan sinyal olarak gönderiliyor,
             yani sadece bu şekli analiz ederek üretecek tamen otomatik bir alet yapman yeterli.
+
     reward_demo_end:
         title: Deneme Sürümünün Sonu
         desc: Deneme sürümünün sonuna geldin!
     reward_balancer:
         title: Dengeleyici
-        desc: Çok işlevli - bütün girdileri eşit olarak bütün çıkışlara dağıtır.!<br><br>
+
+        desc:  Çok fonksioynlu <strong>dengeleyeliyiciyi</strong> açtın!! - daha büyük 
+        fabrikalar yaratmmak için <strong>eşyaları</strong> birden çok konveyorlara ayırıp 
+        birleştirmek için kullanılabilir!<br><br>
```

Comment:
This needs to be indented, it's causing syntax errors. Should be 

```yaml
        desc:  Çok fonksioynlu <strong>dengeleyeliyiciyi</strong> açtın!! - daha büyük 
            fabrikalar yaratmmak için <strong>eşyaları</strong> birden çok konveyorlara ayırıp 
            birleştirmek için kullanılabilir!<br><br>
```

Reply:
Syntax error and misused word "konveyorlar". It was translated as "bant" through the entire translation.

Corrected version is as follows:

        desc:  Çok fonksiyonlu <strong>dengeleyeliyiciyi</strong> açtın!! - daha büyük
            fabrikalar yaratmak için <strong>eşyaları</strong> birden çok bantlara ayırıp
            birleştirmek için kullanılabilir!<br><br>